lib/helpers/multiSigInterfaceHelper.ts

```diff
@@ -0,0 +1,151 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+// @ts-ignore -- not sure why i'm getting error here when compiling, but not in the test files.
```

Comment:
Not sure how to fix this, besides ignoring it. Not even sure why I don't get the same error from `test-ts/*ts` importing the typechain-types.